.gitignore

```diff
@@ -0,0 +1,48 @@
+
+# Created by https://www.gitignore.io/api/linux,macos
+# Edit at https://www.gitignore.io/?templates=linux,macos
+
+### Linux ###
+*~
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
+
+### macOS ###
+# General
+.DS_Store
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
+
+# Thumbnails
+._*
+
+# Files that might appear in the root of a volume
+.DocumentRevisions-V100
+.fseventsd
+.Spotlight-V100
+.TemporaryItems
+.Trashes
+.VolumeIcon.icns
+.com.apple.timemachine.donotpresent
+
+# Directories potentially created on remote AFP share
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+# End of https://www.gitignore.io/api/linux,macos
```

Comment:
I don't think this should be added. This is good for you to add to your own machine as a global .gitignore, but it just clutters up this repo.

Reply:
Oh dang, I've been checking in that .gitignore file to all my repositories. Didn't know global gitignore was possible.